airtable/README.md

```diff
@@ -1,71 +1,81 @@
 ### Overview
```

Comment:
Should this be merged before or after the airtable PR? Or do you want to remove this file from the changes?